comid/measurement_test.go

```diff
@@ -112,3 +123,227 @@ func TestMeasurement_NewUUIDMeasurement_bad_uuid(t *testing.T) {
 
 	assert.Nil(t, tv.SetUUID(nonRFC4122UUID))
 }
+
+var (
+	goldenMKey  uint64 = 100
+	goldenMKey1 uint64 = 120
+)
+
+func TestUIntMkey_NotSet(t *testing.T) {
+	mkey := &Mkey{}
+	expectedErr := "unknown measurement key type: <nil>"
+	err := mkey.Valid()
+	require.NotNil(t, err)
+	assert.EqualError(t, err, expectedErr)
+}
+
+func TestUIntMkey_MarshalCBOR_ok(t *testing.T) {
+	tvs := []struct {
+		mkey     uint64
+		expected []byte
+	}{
+		{
+			mkey:     goldenMKey,
+			expected: ([]byte{0x18, 0x64}),
+		},
+		{
+			mkey:     goldenMKey1,
+			expected: ([]byte{0x18, 0x78}),
+		},
+	}
+
+	for _, tv := range tvs {
+		meas := NewUintMeasurement(tv.mkey)
+		require.NotNil(t, meas)
```

Comment:
I'm not sure why we need to go through the full blown Measurement here?  Is it not sufficient to instantiate an MKey?

Reply:
I thought about this. MKey does not live stand alone, so best use is to use the correct route to set Mkey which is via setting measurements, via newly created api.

Reply:
&mkey{tv.mkey}